apps/myjobs/test/models/configuration_singleton_test.rb

```diff
@@ -65,6 +69,10 @@ def config_via_runner(env: 'development', envvars: '')
   end
 
   test "loading custom AweSim external config in production env" do
+    # TypeError: incompatible marshal file format (can't be read)
+    #     format version 4.8 required; 32.32 given
+    skip
```

Comment:
What's this? I found this [rubygems ticket on for the same](https://github.com/rubygems/rubygems/issues/1589). Could it be an environment issue you have?  I'll pull to replicate too.

Reply:
Scratch that, it likely doesn't like the env file and has nothing to do with rubygems.

I can't replicate on Linux, either on my machine or in pitzer. In any case, I think we should keep it just to safeguard us from future mistakes.

Reply:
So you are saying if you revert https://github.com/OSC/ondemand/pull/702/commits/8d018238d699486db25a09b8593dca0be48a9646 you have no problem?

Reply:
Yep. I reverted on Pitzer and my Linux machine and couldn't get the same error.  Plus Travis in `master` would see it too right?

Reply:
reverted with c7182fba9fbc4a8faeaef9d1fbcd6e8a7b127d67